.evergreen/orchestration/configs/sharded_clusters/basic.json

```diff
@@ -1,30 +1,48 @@
 {
-  "id": "shard_cluster_1",
-  "shards": [
-    {
-      "id": "sh01",
-      "shardParams": {
-        "members": [{
-          "procParams": {
+    "id": "shard_cluster_1",
+    "shards": [
+        {
+            "id": "sh01",
+            "shardParams": {
+                "members": [
```

Comment:
Your OP says:

> In all cases, we test against standalone, replica set, and sharded cluster topologies. The latter uses a replica set by default, but we override the provided file to make sure the replica set has secondaries, which is required for some tests.

It's been a while since I looked an MO config for sharding, but I assume config servers aren't specified at all? In that case, I assume `members` here comprises the replica set for the first shard, and two of these three nodes will end up as secondaries. Correct?

Would it make sense to rename this file to `sharded-rs.json` instead of `basic.json`?

Reply:
I can rename it to `sharded-rs.json` and use a different `ORCHESTRATION_FILE` config in evergreen. The default `basic.yml` already specifies a replica set as shard, albeit with a single server which causes one test to fail as it attempts to select a secondary.

Reply:
Oh, I missed that the original was a single-node replica set. I'm fine leaving this as-is, then.

I don't imagine we'll actually need a test environment for a shard with a single-node RS or standalone.